homework/06LessonSix.js

```diff
@@ -98,6 +114,12 @@ var conferenceRoom = new ConferenceRoom();
 // What is a method?
 //
 //
+// A method is a function created in an Object Constructor. Methods can be generalized an apply to all instances of
+// a class or methods can be individualized, specified, and unique so that only a certain instance of the class are able to per
+// If human is a class then walking is a method that most humans have.
+// marySol is also part of the human class. She has the walk() method at her disposal. However, because marySol is unique she has
+// certain methods other humans do not have. Each day she uses method codePearl() while most humans not only do not do this but they CANNOT do this.
```

Comment:
Hahahahah!!! As is understandable...No human in their right mind should be invoking codePearl()! X_X

Manny.greatWork();

![good](http://stream1.gifsoup.com/view5/2610352/good-job-o.gif)